js/src/personal/jionghao/components/Experience.module.css

```diff
@@ -6,11 +6,29 @@
 }
 
 .verticalTab{
-  padding-right: 4vw;
-  padding-top: 2vh;
-  padding-bottom: 2vh;
+  padding-right: 3vw;
```

Comment:
Stick with bigger numbers if you're using vh/vw. 
If it's this small, use rem/px